src/System.Private.Xml.Linq/src/System/Xml/Linq/XObjectChangeAnnotation.cs

```diff
@@ -8,9 +8,5 @@ internal class XObjectChangeAnnotation
     {
         internal EventHandler<XObjectChangeEventArgs> changing;
         internal EventHandler<XObjectChangeEventArgs> changed;
-
-        public XObjectChangeAnnotation()
-        {
-        }
```

Comment:
This isn't dead code per se, it's just that the compiler will emit the exact same thing if it's not there.

Reply:
Do we ourselves generally bother writing parameterless public constructors? If not, it seems reasonable to remove them..

Reply:
> Do we ourselves generally bother writing parameterless public constructors?

It depends.  We write them if there's any other constructor, as the presence of any constructor will prevent the compiler from adding the parameterless one.  Or if we want to change the visibility.

In this case, it's ok removing it, I was just calling out that whether it's written here or not, it'll still end up in the binary.

Reply:
> We write them if there's any other constructor, as the presence of any constructor will prevent the compiler from adding the parameterless one.

Yes, of course I should have said that explictily 👿